src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+    >
```

Comment:
The form tag is incorrectly self-closed. In HTML5, form elements should not be self-closing. Please remove the '/' at the end of the form tag and ensure it has a proper closing tag.